string agv_pairs
uint8 error_id
string time_collision